spec/readthis/cache_spec.rb

```diff
@@ -144,6 +144,11 @@
         'e' => 2,
       )
     end
+
+    it 'returns {} with no keys' do
+      expect(cache.read_multi(namespace: 'cache')).to eq({})
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.